front/app/containers/Admin/projects/project/analysis/InputsList/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import useInfiniteInputs from 'api/analysis_inputs/useInfiniteAnalysisInputs';
+import { useSearchParams, useParams } from 'react-router-dom';
+import T from 'components/T';
+import Button from 'components/UI/Button';
+import { pick } from 'lodash-es';
+const InputsList = () => {
+  const { analysisId } = useParams() as { analysisId: string };
+  const [searchParams] = useSearchParams();
+
+  const allParams = Object.fromEntries(searchParams.entries());
+  const filters = pick(allParams, [
+    'search',
+    'published_at_from',
+    'published_at_to',
+    'reactions_from',
+    'reactions_to',
+    'votes_from',
+    'votes_to',
+    'comments_from',
+    'comments_to',
+  ]);
+
+  const { data, fetchNextPage, hasNextPage } = useInfiniteInputs(
```

Comment:
Hooks should accept only one argument that is a single object. It's related to the way we construct the key.